temporal-sdk/src/main/java/io/temporal/testing/TestEnvironmentOptions.java

```diff
@@ -77,29 +83,52 @@ public Builder setMetricsScope(Scope metricsScope) {
       return this;
     }
 
+    public Builder setUseExternalService(boolean useExternalService) {
```

Comment:
rename to target and add appropriate JavaDoc.

Reply:
Do you want to rename to `setUseExternalTarget`? Added a javadoc.

Reply:
Addressed the naming in `WorkerFactoryOptions`